docs/daemon/setup/setup.mdx

```diff
@@ -0,0 +1,160 @@
+
+You also have the option to modify some default settings of the genesis file using below flags:
+- `--without_gas_costs`
+- `--without_nonces`
+- `--validator_join_expiry`
+- `--chain_id_prefix`
+
+However, these flags are only effective when generating a new genesis.json file. If a genesis.json file already exists, the node will operate based on its current configuration
+
+### Setup using the kwil-admin cli
+
+The root directory can be initialized with the default configuration files using the kwil-admin command line utility as shown below.
+
+```bash
+kwil-admin setup init -o "path/to/root/dir"
+```
+This command will generate the missing private_key, config.toml, genesis.json files using default settings. 
+
+### Manual configuration
+Manually generate and configure private_key, genesis.json and config.toml as shown below.
+
+#### Generate private_key
+Generate private key file using the kwil-admin utils as shown below.
+```bash
+kwil-admin key gen -o "path/to/private/key/file"
+```
+
+#### View key information
+Public Key, NodeID, Address corresponding to the private key can be viewed using the below command. Public key and Address are required to configure genesis file and nodeID is required to populate persistent_peers
+```bash
+kwil-admin key info -k "path/to/private/key/file"
+
+Private key (hex): 132fd675ea871170a21e2b0f99fa68cbb99ccefb3ff7f2a38856105caf101d1af2003e9021ba911b58c1a6da0aef211a038d343f2d3a95db7bc3dca7805b939b
+Private key (base64): Ey/WdeqHEXCiHisPmfpoy7mczvs/9/KjiFYQXK8QHRryAD6QIbqRG1jBptoK7yEaA400Py06ldt7w9yngFuTmw==
+Public key (base64): 8gA+kCG6kRtYwabaCu8hGgONND8tOpXbe8Pcp4Bbk5s=
+Public key (cometized hex): PubKeyEd25519{F2003E9021BA911B58C1A6DA0AEF211A038D343F2D3A95DB7BC3DCA7805B939B}
+Public key (plain hex): f2003e9021ba911b58c1a6da0aef211a038d343f2d3a95db7bc3dca7805b939b
+Address (string): 494317B995E6A4522E5610183DEC52A33F034B0A
+Node ID: 494317b995e6a4522e5610183dec52a33f034b0a
+```
+
+#### Genesis file configuration
+Copy the sample [genesis.json](./../configuration/genesis.mdx#complete-genesis-file) into `root_dir/abci/config/genesis.json` and update the configuration as required.
+
+Manually update the chain_id, initial_height, app_hash configurations in the genesis.json file.
+The current node can be added as one of the inital validators in the genesis file as below. The base64 public key is used here. 
+```bash
+  "validators": [
+    {
+      "address": "494317B995E6A4522E5610183DEC52A33F034B0A",
+      "pub_key": {
+        "type": "tendermint/PubKeyEd25519",
+        "value": "8gA+kCG6kRtYwabaCu8hGgONND8tOpXbe8Pcp4Bbk5s="
+      },
+      "power": "1",
+      "name": "validator-0"
+    }
+  ],
+```
+#### Config.toml 
+Download the sample [config.toml](./../configuration/default_config.toml) into `root_dir/config.toml` and update the configuration as required. The minial configuration required is: 
+
+```bash
```

Comment:
No bash for this code block.  Actually, I don't think this is a code block with the plain text headers and bullets lists, but nbd.